spec/system/dashboard_system_spec.rb

```diff
@@ -162,8 +162,9 @@
         ["Last 30 Days", test_time - 29.days,                     test_time],
         ["This Month",   test_time.beginning_of_month,            test_time.end_of_month],
         ["Last Month",   test_time.last_month.beginning_of_month, test_time.last_month.end_of_month],
-        ["This Year",    test_time.beginning_of_year,             test_time.end_of_year],
-        ["All Time",     test_time - 100.years,                   test_time]
+        ["This Year",    test_time.beginning_of_year,             test_time.end_of_year]
+      #  We now can't test the lower limit of All Time, because the earliest possible date is 2000-01-01
+      #  ["All Time",     test_time - 100.years,                   test_time]
```

Comment:
Can't you test it by changing this to "2001-01-01" instead of 100 years ago?

Reply:
IIRC,  there is something in the whole sequence that checks by making a distribution that is before the timeframe and seeing if doesn't appear.   That would break now if we leave this check in.   Even if we make all time start at Jan 1 2000.

Reply:
We can still test this functionality - "All Time" means since 2000, so we should have one distribution in 1999 and one in 2001 and we should validate that the second is included and the first is not.

Reply:
Except that we no longer allow distributions in 1999

Reply:
Those are 2 different validations - we can violate one to test the other.

Reply:
I may be missing something.  My understanding is that the way the system-level tests are set up,  it would try to create a 1999-12-31 donation to test that the "All Time" filter works.   The creation of that donation is going to fail, and hence the test would fail.

Reply:
Ah right! We'd have to manually bypass validations to check this, which I think is a mite too far. Should be fine without it.